api/types/matchers_aws.go

```diff
@@ -274,3 +281,35 @@ func (m *AWSMatcher) CheckAndSetDefaults() error {
 	}
 	return nil
 }
+
+// HasOrganizationMatcher returns true if the matcher has an organization ID set.
+func (m *AWSMatcher) HasOrganizationMatcher() bool {
+	return m.Organization != nil && m.Organization.OrganizationID != ""
+}
+
+func (m *AWSMatcher) validOrganizationAccountDiscovery() error {
```

Comment:
I think it makes sense to use a verb here, based on name alone "valid" sounds like it should return boolean.


```suggestion
func (m *AWSMatcher) validateOrganizationAccountDiscovery() error {
```